### ISSUE TITLE
Question about loss function

### ISSUE THREAD

**Issue by littlewhitesea** [YapengTian/TDAN-VSR-CVPR-2020#26]
Thanks for your excellent work. I have some questions about loss function.

1. I found the loss function is MSE Loss, https://github.com/YapengTian/TDAN-VSR-CVPR-2020/blob/7d0418a858fbfce201fba98929bcbfb48064bf67/solver.py#L179 but in the paper, the objective function of the network is defined via L1 reconstruction loss. Does it mean MSE Loss is better than L1 loss for video super-resolution?

2. https://github.com/YapengTian/TDAN-VSR-CVPR-2020/blob/7d0418a858fbfce201fba98929bcbfb48064bf67/solver.py#L223 I am curious about the hyperparameter '0.25'. Could you please tell me more about how to get the appropriate value of hyperparameter during experiments?

Could you help me solve above issues?

**Comment by YapengTian** [comment]
Thanks for your interest! 
(1) I found MSE achieves better performance in our experiments. So, I modified it to MSE. I updated it in our [CVPR](http://openaccess.thecvf.com/content_CVPR_2020/papers/Tian_TDAN_Temporally-Deformable_Alignment_Network_for_Video_Super-Resolution_CVPR_2020_paper.pdf) version.
(2) Since 4 supporting frames are used, 1/4 is used. I just empirically set this hyperparameter in the experiments.

Thanks!

**Comment by littlewhitesea** [comment]
It's so pleasure to receive your response.

(1) I think I downloaded the former version.
(2) Oh, I forgot the intrinsic coefficient of aligned loss and misunderstood 1/4 is a hyperparameter.

Thanks for your quick and detailed reply!
